src/main/java/org/skriptlang/skript/elements/expressions/displays/ExprDisplayGlowOverride.java

```diff
@@ -0,0 +1,90 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.elements.expressions.displays;
+
+import org.bukkit.entity.Display;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Color;
+import ch.njol.skript.util.ColorRGB;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Display Glow Color Override")
+@Description({
+	"Returns or changes the glowing color override of <a href='classes.html#display'>displays</a>.",
+	"This overrides whatever color is already presently set for the scoreboard of the displays."
```

Comment:
Choose `already` or `presently`, but not both together. (or even `already present`, without `set`!)
Also not sure about the usage of `scoreboard` here.

Reply:
The whole point is to explain that it'll override the scoreboard colour setting.

Reply:
I'd suggest using `team`, or at least `scoreboard team`. Since team and scoreboard are two different commands, I didn't connect the two in my head and I'm willing to bet others might do the same.